lib/src/main/java/com/payone/commerce/platform/lib/endpoints/BaseApiClient.java

```diff
@@ -34,7 +34,7 @@ public class BaseApiClient {
 
     protected static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
 
-    private final OkHttpClient client = new OkHttpClient();
+    private OkHttpClient clientSpecificHttpClient;
```

Comment:
What is the idea of having the client here and in the `CommunicatorConfiguration` ? Isn't it enough to have it always in the config with the default as before?

Reply:
Good catch. Actually we had the default value in the BaseApiClient before. But now that we have it in the ConfigurationObject, we can drop it in the BaseApiClient